src/Sylius/Bundle/LocaleBundle/Twig/LocaleExtension.php

```diff
@@ -50,6 +91,52 @@ public function getCurrentLocale()
     }
 
     /**
+     * Returns current language direction.
+     *
+     * @return string Possible values are `rtl` and `ltr`
+     */
+    public function getCurrentDirection()
+    {
+        return $this->helper->getCurrentDirection();
+    }
+
+    /**
+     * Checks whether if current language direction is RTL or not.
+     *
+     * @return bool True if language direction is RTL
+     */
+    public function isRtl()
+    {
+        return $this->helper->getCurrentDirection() === 'rtl';
+    }
+
+    /**
+     * Twig filter to format date in a localized calendar-aware fashion.
+     *
+     * @param \Twig_Environment $env
+     * @param $date
+     * @param $timezone
+     * @param $format
+     *
+     * @return bool|string
+     * @throws \Twig_Error_Runtime
+     */
+    public function filterLocalizedDate(\Twig_Environment $env, $date, $format = null, $timezone = null)
+    {
+        if (null === $format) {
+            $formats = $env->getExtension('core')->getDateFormat();
+            $format = $date instanceof \DateInterval ? $formats[1] : $formats[0];
+        }
+
+        // Determine the timezone
+        if (!$timezone) {
```

Comment:
It is same check as above, so it is inconsistency. Use same condtition as above.